src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -124,7 +124,12 @@ function WorkspacesListPage() {
     const workspaceAccountID = PolicyUtils.getWorkspaceAccountID(policyIDToDelete ?? '-1');
     const [cardFeeds] = useOnyx(`${ONYXKEYS.COLLECTION.SHARED_NVP_PRIVATE_DOMAIN_MEMBER}${workspaceAccountID}`);
     const [cardsList] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${workspaceAccountID}_${CONST.EXPENSIFY_CARD.BANK}`);
-    const hasCardFeedOrExpensifyCard = !isEmptyObject(cardFeeds) || !isEmptyObject(cardsList);
+    const hasCardFeedOrExpensifyCard =
+        !isEmptyObject(cardFeeds) ||
+        !isEmptyObject(cardsList) ||
```

Comment:
```suggestion
```
We don't need these conditions now, validating the cardsEnabled and workspaceAccountID is enough.

Reply:
If the feature is disabled then I don't think it's needed. Let's confirm with @mountiny

Reply:
I think we can keep them in